build/updatehistorictzdb.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+rm -rf tmp
+mkdir tmp
+
+# Get the list of files, just once.
+
+mkdir -p tmp/releases
+cd tmp/releases
+FILES=files.txt
+
+gsutil.cmd ls gs://nodatime/releases > $FILES
+
+# Work out the current release, fetch and extract it
+echo "Fetching and expanding release ${RELEASE}"
+RELEASE=`cat $FILES | grep -o -E 'NodaTime-1\.3\.[0-9]+\.zip' | sort -r | head -n 1 | sed s/NodaTime-// | sed s/.zip//`
+gsutil.cmd cp gs://nodatime/releases/NodaTime-${RELEASE}.zip .
+gsutil.cmd cp gs://nodatime/releases/NodaTime-${RELEASE}-src.zip .
+curl -s -S https://api.nuget.org/packages/nodatime.${RELEASE}.nupkg -o nodatime.${RELEASE}.nupkg
+unzip -q NodaTime-${RELEASE}.zip
+unzip -q NodaTime-${RELEASE}-src.zip
+unzip -q nodatime.${RELEASE}.nupkg -d NodaTime-${RELEASE}-nupkg
+
+# Handy "increment version number" code from http://stackoverflow.com/questions/8653126
+NEW_RELEASE=`echo $RELEASE | perl -pe 's/^((\d+\.)*)(\d+)(.*)$/$1.($3+1).$4/e'`
```

Comment:
nit: prefer

```sh
$(foo)
```

over

```sh
`foo`
```

(it has saner nesting behaviour, if nothing else)